tests/test_core.py

```diff
@@ -136,513 +133,3 @@ def test_get_spent_issue_1271(b, alice, bob, carol):
     assert not b.get_spent(tx_5.id, 0)
     assert b.get_outputs_filtered(alice.public_key)
     assert b.get_outputs_filtered(alice.public_key, spent=False)
-
-def encode_tx_to_bytes(transaction):
```

Comment:
@andreitricolici why did these test cases got removed?

Reply:
@LaurentDeMontBlanc as i write in description. function from tests/test_core.py are duplicates from tests/tendermint/test_core.py